crates/loader/tests/ui.rs

```diff
@@ -0,0 +1,65 @@
+use std::{ffi::OsStr, path::Path};
+
+use futures::Future;
+use spin_loader::cache::Cache;
+use ui_testing::{Failed, Normalizer, UiTestsRunner};
+
+fn main() -> anyhow::Result<()> {
+    // Insert dummy wasm into cache to avoid network traffic
```

Comment:
I'm confused by this. Why would the network be hit if we don't add the dummy?

Reply:
`valid-manifest.toml` has a remote source which the loader would (try and fail to) download.